csr2transmart/mappers/ontology_mapper.py

```diff
@@ -49,8 +49,9 @@ def map_concept_node(self, node: OntologyConfigTreeNode) -> ConceptNode:
         concept = Concept(node.concept_code, node.name, concept_path, concept_type)
         self.concept_code_to_concept[node.concept_code] = concept
 
-        metadata_value: Dict[str, str] = {
-            'subject_dimension': self.entity_name_to_subject_dimension_value(entity_name)}
+        metadata_value: Dict[str, str] = dict()
+        if entity_name in list(map(lambda se: se.schema()['title'], SubjectEntity.__args__)):
```

Comment:
The list of entity names can also be computed once at the top of the file, instead of recomputing it for every node.